### PR TITLE
fix: 🐛 Fix Table filter menu have no height

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close ant-design/ant-design#23166

### 💡 Background and solution

when less version is 2.x

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix Table filter menu no height when less version is 2.x. |
| 🇨🇳 Chinese | 修复 Table 筛选菜单在 less 版本为 2.x 时不显示的问题。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/table/style/index.less [hunk 1]
```diff
@@ -447,7 +447,7 @@
     .@{ant-prefix}-dropdown-menu {
       // https://github.com/ant-design/ant-design/issues/4916
       // https://github.com/ant-design/ant-design/issues/19542
-      max-height: calc(100vh - 130px);
+      max-height: ~'calc(100vh - 130px)';
```
Comment (afc163) [inline components/table/style/index.less:450]: less 2 不支持上面的写法。

Comment (yoyo837) [reply components/table/style/index.less:450]: `max-height: calc(~'100vh - 130px');` 这种是不是也是less 2支持的？

Comment (afc163) [reply components/table/style/index.less:450]: 不了解，可以试试。

Comment (yoyo837) [reply components/table/style/index.less:450]: 试了下好像都可以，less 2/3也都支持这种

Comment (afc163) [reply components/table/style/index.less:450]: 本质上和当前的改法是一样的。

**Pull request merged**